packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -318,7 +324,7 @@ export class Builder extends EventEmitter {
       throw new Error(`No project found for file: ${filePath}`);
     }
 
-    const projectName = project.projectName || '';
+    const projectName = project.resolveProjectName('');
```

Comment:
Do you still need the empty string argument?

Reply:
yup, the empty string is necessary since we want the `projectName` to be falsy when we evaluate how to construct the schema types file name.

Reply:
No one but you is ever going to remember why this is the case, and even you'll forget soon enough. We either need to come up with a more elegant solution or this needs a bunch of comments.

Reply:
it looks like `Map<K, V>` supports `undefined` as a key, so i don't need to do any of this non-sense.